code401-challenges/src/test/java/code401/challenges/FizzBuzzTree/FizzBuzzTreeTest.java

```diff
@@ -0,0 +1,32 @@
+package code401.challenges.FizzBuzzTree;
+
+import code401.challenges.tree.Node;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class FizzBuzzTreeTest {
```

Comment:
It would have been a good idea to test and make sure the your method worked on the entire tree, not just on the root node!